### DISCUSSION TITLE
How to create a clone of a project/DB?

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#29409] [Questions] [answered]
Hi,  I am fairly new to supabase so bear with me, in ms-sql server I could just make a backup and restore to a new DB in order to clone a DB, but how do I do that in Supabase? is there a straight forward way, or if not is there a guide?

**Comment by GaryAustin1** [comment, marked as answer]
Postgres has pg_dump for the database.  The Supabase CLI has tools that use that,or you can use native tools like PSQL.
https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#migrate-your-project

Possibly related depending on what you want to do:  https://supabase.com/docs/guides/platform/branching
